contracts/standard/arbitration/AppealableArbitrator.sol

```diff
@@ -1 +1,58 @@
-// TODO: Create a class allowing appeal to another arbitrator.
+/**
+ *  @title Arbitration Standard 
+ *  @author Mahadevan K - <mahadevan.k@gmail.com>
+ *  Bug Bounties: This code hasn't undertaken a bug bounty program yet.
+ */
+
+pragma solidity ^0.4.15;
+
+import "./CentralizedArbitrator.sol";
+
+/** @title Appealable Arbitrator
+ *  This is a single arbitrator deciding alone of the result of disputes, but appeals are possible unlike in CentralizedArbitrator
+ */
+contract AppealableArbitrator is CentralizedArbitrator {
+    /** @dev Cost of appeal, as of now just multiplying this by two to create increasing cost for appeals
+     *  @param _disputeID ID of the dispute to be appealed. Not used by this contract.
+     *  @param _extraData Not used by this contract.
+     *  @return fee Amount to be paid.
+     */
+    function appealCost(uint _disputeID, bytes _extraData) public constant returns(uint fee) {
+        return arbitrationPrice*2;
+    }
+
+    /**
+     * @dev - remove owner as arbitrator since appeal was successful, this function assumes that the arbitrable is reassigned by Arbitrable
+     * @param _disputeID ID of the dispute to remove
+     */
+    function appealSuccessful(uint _disputeID) public {
```

Comment:
If the dispute is appealable, the function just deletes the dispute.
It should transfer it to another Arbitrable contract.